### DISCUSSION TITLE
有没有监听群成员修改昵称的事件

### DISCUSSION THREAD

**Discussion by yujumpjump** [simple-robot/Discussions#85] [Q&A] [answered]
### 涉及的编程语言

Java

### 涉及的组件库

其他

### 疑问描述

怎么来监听群成员修改了群昵称 或者说不让他修改自己的昵称

**Comment by ForteScarlet** [comment, marked as answer]
https://docs.simbot.forte.love/components/mirai/simbot-component-mirai-core/love.forte.simbot.component.mirai.event/-mirai-member-card-change-event/index.html

**Reply by yujumpjump** [reply]
我看了一下 这个必须在他修改昵称后 在群里面在发送一条消息后才可以监听到 。有没有那种刚修改就发现了的呢

**Reply by ForteScarlet** [reply]
没有
